### ISSUE TITLE
Doesn't detect duplicate keys

### ISSUE THREAD

**Issue by alexjurkiewicz** [Pryz/yaml-lint#11]
Hi, it would be great if the following YAML document would raise an error:

``` yaml
key:
  val: 1
  val: 2
```

You can see this rule implemented in `yamllint`: http://yamllint.readthedocs.io/en/latest/rules.html#module-yamllint.rules.key_duplicates

**Comment by Pryz** [comment]
Totally agree. Will try to work on it asap ;)

**Comment by alexharv074** [comment]
@Pryz , is this project still maintained, and if it is, how hard would it be to implement this feature, and if not too hard, would you accept a PR that implements it?

**Comment by Pryz** [comment]
Unfortunately I don't have much time to work on this project anymore. However I will totally be in for a PR :)

From the doc shared by @alexjurkiewicz it seems pretty easy to implement this feature.
